core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1525,10 +1539,27 @@ private[spark] class DAGScheduler(
             s" ${task.stageAttemptId} and there is a more recent attempt for that stage " +
             s"(attempt ${failedStage.latestInfo.attemptNumber}) running")
         } else {
+          // Gracefully handling the stage abort due to fetch failure in the
+          // decommission nodes
+          if (!event.reason.asInstanceOf[FetchFailed].countTowardsStageFailures) {
+            // Ignore stage attempts due to fetch failed only
```

Comment:
I think comment should specifically say for decommission tracking.

Reply:
done